pkgs/servers/gemini/agate/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, rustPlatform, installShellFiles }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "agate";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "mbrubeck";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-rwoEZnxh0x+xaggJuoeSjE1ctF43ChW5awcDJyoWioA=";
```

Comment:
```suggestion
    hash = "sha256-rwoEZnxh0x+xaggJuoeSjE1ctF43ChW5awcDJyoWioA=";
```

Reply:
When did this change?

Reply:
Both variants work, but since you are already using SRI hash (`sha256-...`), why not using `hash` instead of legacy `sha256`.

Reply:
Ok so going forward all should be hash?
Also this is backwards compatible with non-2.4pre versions of nix?

Reply:
SRI hashes technically should use hash but older nix versions do not support it. Right now it is whatever and will maybe change in the future.

Reply:
Im only using the SRI hashes because thats what my nix version gives me and they seem to be backwards compatible

Reply:
> Also this is backwards compatible with non-2.4pre versions of nix?

Yes, it works with nix 2.3.